packages/atomic/src/components/facets/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -72,19 +79,29 @@ export class AtomicDateFacet implements InitializableComponent, BaseFacetState {
    * The format that the date will be displayed in. See https://day.js.org/docs/en/display/format for formatting details.
    */
   @Prop() public dateFormat = 'DD/MM/YYYY';
+  /**
+   * The number of values to request for this facet, when there are no manual ranges.
+   */
+  @Prop({mutable: true}) public numberOfValues = 10;
```

Comment:
I was looking at Atomic recently, and I felt a default of 10 values on the specific facet, and now the date facet, was quite high. Wasn't the UX recommendation 8 values? Or did the recommendation change?

Example: https://static.cloud.coveo.com/searchui/v2.10084/index.html#t=All&sort=relevancy

Reply:
It is pretty high yes, I don't know where this changed, I think it was a decision done during facet's implementation, has no UX work was done prior